services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/attachment/taydennys/TaydennysAttachmentControllerITest.kt

```diff
@@ -63,6 +77,61 @@ class TaydennysAttachmentControllerITest(@Autowired override val mockMvc: MockMv
         confirmVerified(attachmentService, authorizer)
     }
 
+    @Nested
+    inner class GetAttachmentContent {
+        @Test
+        fun `when valid request should return attachment file`() {
```

Comment:
Since the time the application attachment controller tests were written, we've settled on a naming convention for the tests. It's been established as `expected behaviour when state under test`, where the when-part can be left out if the state is the assumed happy path.

I don't think it's worth the effort to change all existing tests to follow this convention, we should use it for any new tests, even when they are copy-pasted from older tests.

Reply:
I changed the names of the tests.